extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/methods/StandardMethodImplementor.java

```diff
@@ -144,6 +149,21 @@ protected void addContextAnnotation(AnnotatedElement element) {
         element.addAnnotation(Context.class);
     }
 
+    protected void addOpenApiAnnotations(AnnotatedElement element, String returnType, Response.Status status) {
+        if (isOpenApiEnabled) {
+            try {
+                element.addAnnotation(APIResponseSchema.class)
+                        .add("value", Thread.currentThread().getContextClassLoader().loadClass(returnType))
+                        .add("responseCode", String.valueOf(status.getStatusCode()));
+
+                // We need to add an empty APIResponses annotation, so OpenAPI does not try to determine the return type.
+                element.addAnnotation(APIResponses.class.getName());
+            } catch (ClassNotFoundException ex) {
+                LOGGER.error("Unable to add the @APIResponseSchema annotation", ex);
```

Comment:
What can the user do about this? If the user cannot do anything about it and it's just an invalid state, then it should probably an `IllegalStateException`

Reply:
This should never happen. But I agree with you and will change this and [this](https://github.com/quarkusio/quarkus/blob/f664de6c0fe62e8cbc61bc14ccd95045d583f73d/extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/methods/StandardMethodImplementor.java#L111) to avoid hiding these exceptions.